### PR TITLE
Clarify what constitutes a valid license

### PR DESCRIPTION
**Pull request by andreasohlund**
@Particular/docs-maintainers is partials with major.minor range supported? 

@Particular/nservicebus-maintainers please review

### CODE REVIEW

File: nservicebus/licensing/index.md [hunk 1]
```diff
@@ -21,6 +21,9 @@ related:
 
 See [Particular Licensing](https://particular.net/licensing) for license specifics.
 
+## Is my license valid?
```
Comment (SimonCropp) [inline nservicebus/licensing/index.md]: remove the personal voice

File: nservicebus/licensing/index_validity_core_[3,4.5).partial.md [hunk 2]
```diff
@@ -0,0 +1 @@
+The license is valid if the `LicenseVersion` attribute in the license data is greater than or equal to the `{major}.{minor}` part of the fileversion for the NServiceBus assembly used by the endpoint.
```
Comment (SimonCropp) [inline nservicebus/licensing/index_validity_core_[3,4.5).partial.md]: @andreasohlund unfortunately partials can only match what the version switch will allow. in this case i recommend u have  one partial targeting "5 and lower" with version specific language in it. and then one for "6 and higher"

Comment (andreasohlund) [inline nservicebus/licensing/index_validity_core_[3,4.5).partial.md]: Ok, will rearrange the partials

File: nservicebus/licensing/index.md (hunk 1 above)
Comment (andreasohlund) [inline nservicebus/licensing/index.md]: How about "License validity" ?

Comment (SimonCropp) [inline nservicebus/licensing/index.md]: yep

**Comment by andreasohlund** [comment]
This one should be good to go now

**Comment by SimonCropp** [comment]
Looks like the build is failing

**Comment by andreasohlund** [comment]
Property "community project" not found? (I'll try to rebase and see if it goes away)

**Comment by SimonCropp** [comment]
Hmmm. Is master building?  I might have broken it

**Comment by andreasohlund** [comment]
Master is green, I'll digg deeper

**Comment by andreasohlund** [comment]
SerializationException : Property 'IsCommunityProject' not found on type 'SerializationComponent'.

Seems unrelated to the changes I've made?

**Comment by SimonCropp** [comment]
Yeah leave it with me.

**Pull request merged**